setup.py

```diff
@@ -49,37 +35,34 @@
 
 setup(
 	name='EAPeak',
-	version='0.1.4',
+	version='0.1.5',
```

Comment:
This should actually be `0.1.6` like it is in eapeak. I'm guessing this file has been one version behind for a while.